test/test_torch.py

```diff
@@ -1695,7 +1695,10 @@ def test_add(self):
         m2 = torch.tensor([True, True, False, False, False, True], dtype=torch.bool)
         self.assertEqual(m1 + m2, torch.tensor([True, True, False, True, False, True], dtype=torch.bool))
 
-        # [res] torch.add([res,] tensor1, value, tensor2)
+        # bfloat16
+        m1 = torch.tensor([1, 2], dtype=torch.bfloat16)
```

Comment:
these should be e.g. `1., 2.` to indicate you mean for them to be floating point.